frontend/packages/component-library/src/spacer/__tests__/Spacer.test.tsx

```diff
@@ -0,0 +1,19 @@
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Spacer, {SpacerProps} from '../Spacer';
```

Comment:
nit: for consistency would be nice to update it to
```suggestion
import Spacer, {SpacerProps} from './../Spacer';
```